features/step_definitions/navigation.rb

```diff
@@ -1,26 +1,30 @@
+def current_user
```

Comment:
I'm not sure a global function like this is a great idea but we can worry about that later